server/views/assessments/pages/make-a-decision/make-a-decision.njk

```diff
@@ -37,4 +37,18 @@
 
       {%endcall%}
     </div>
+    {{
+        formPageTextarea(
+          {
+            fieldName: "rejectReason",
+            type: "textarea",
+            spellcheck: false,
+            label: {
+              text: "Reject reason",
```

Comment:
```suggestion
              text: "Reason for rejection",
```

Reply:
or perhaps "Rationale for your decision"